### ISSUE TITLE
could you provide the full experiment settings for all the datasets?

### ISSUE THREAD

**Issue by mjqgit** [xzr12/PredCNN#4]
### Could you please provide the full hyper-parameters settings and the data processs for TaxiBJ and other datasets in your experiment? I want to reproduce your results but I am not very clear when I read the papers.
And another question is that, why the TaxiBJ experiment results are so different in this work and the other work of your team(**Memory In Memory: A Predictive Neural Network for Learning Higher-Order Non-Stationarity from Spatiotemporal Dynamics**). One of the metric  is RMSE while the other is MSE, but the square of RMSE is also not equal to MSE.(see the PredRnn result of this two work)
![image](https://user-images.githubusercontent.com/28883713/58153452-98fff180-7ca1-11e9-898b-106352a97f1c.png)
![image](https://user-images.githubusercontent.com/28883713/58153883-d153ff80-7ca2-11e9-955c-306366f3786a.png)
